src/Microsoft.Data.SqlClient/tests/FunctionalTests/GenericConverterTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlTypes;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
```

Comment:
Please, cover at least all the conversion with `primitive types` and especially `object`.

Reply:
done -- however i commented out the code because there currently is not an easy way to reference this file according to your preferences and the current project setup.

Any feedback is appreciated!